packages/svelte/src/internal/client/dom/elements/attributes.js

```diff
@@ -241,14 +241,20 @@ var setters_cache = new Map();
 function get_setters(element) {
 	/** @type {string[]} */
 	var setters = [];
+	var descriptors;
+	var proto = get_prototype_of(element);
 
-	// @ts-expect-error
-	var descriptors = get_descriptors(element.__proto__);
+	// Stop at Element, from there on there's only unnecessary setters we're not interested in
+	while (proto.constructor.name !== 'Element') {
```

Comment:
Will this work with SVG elements?

Reply:
Good question - I checked and `Element` is also part of the prototype chain, so it works.